### PR TITLE
`spice add` automatically creates spicepod.yaml if it does not exist

### PR DESCRIPTION
**Pull request by lukekim**
Uses the cwd name.

### CODE REVIEW

File: bin/spice/pkg/spicepod/spicepod.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		Kind:    "Spicepod",
+	}
+
+	skeletonPodContentBytes, err := yaml.Marshal(skeletonPod)
+	if err != nil {
+		return "", fmt.Errorf("Error marshalling spicepod: %w", err)
+	}
+
+	spicepodPath := path.Join(spicepodDir, "spicepod.yaml")
+	err = os.WriteFile(spicepodPath, skeletonPodContentBytes, 0766)
```
Comment (y-f-u) [inline bin/spice/pkg/spicepod/spicepod.go]: 744 might make more sense.

Comment (lukekim) [inline bin/spice/pkg/spicepod/spicepod.go]:
```suggestion
	err = os.WriteFile(spicepodPath, skeletonPodContentBytes, 0744)
```

File: bin/spice/pkg/spicepod/spicepod.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+	"github.com/spiceai/spiceai/bin/spice/pkg/api"
+	"gopkg.in/yaml.v2"
+)
+
+func CreateManifest(name string, spicepodDir string) (string, error) {
+	fs, err := os.Stat(name)
+	if err != nil {
+		if os.IsNotExist(err) {
+			err = os.Mkdir(name, 0766)
```
Comment (phillipleblanc) [inline bin/spice/pkg/spicepod/spicepod.go:16]:
```suggestion
			err = os.Mkdir(name, mode.Perm)
```

Comment (y-f-u) [inline bin/spice/pkg/spicepod/spicepod.go:16]: this is probably 755.

**Review by y-f-u — APPROVED** [review]

**Pull request merged**